management/src/clusterctl/main.go

```diff
@@ -21,11 +21,22 @@ var (
 		},
 	}
 
-	extraVarsFlags = []cli.Flag{
+	extraVarsStringFlag = cli.StringFlag{
+		Name:  "extra-vars, e",
+		Value: "",
+		Usage: "extra vars for ansible configuration. This should be a quoted json string.",
+	}
+
+	cmdFlags = []cli.Flag{
+		extraVarsStringFlag,
+	}
+
+	commissionFlags = []cli.Flag{
+		extraVarsStringFlag,
 		cli.StringFlag{
-			Name:  "extra-vars, e",
+			Name:  "host-group, g",
 			Value: "",
-			Usage: "extra vars for ansible configuration. This should be a quoted json string.",
+			Usage: "host-group. service-master, service-worker",
```

Comment:
I think usage string can be a bit more verbose.

One possibility:

`host-group of the node(s). Possible values: service-master or service-worker`